CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
+- Fix several issues in Markdown CmdLets (#7329)
+- Add support for parsing Link Header with variable whitespace (#7322)
+- Change parameters order in `Get-Help` and help in order to get first `-Full` and
+  then `-Functionality` when using Get-Help `-Fu␉` and help `-Fu␉` (#7370) (Thanks @sethvs!)
+- Add support for passing files and markdown directly to `Show-Markdown` (#7354)
+- Add `-SkipIndex` parameter to `Select-Object` (#7483) (Thanks @powercode!)
+- Improve performance of `Import-CSV` up to 10 times (#7413) (Thanks @powercode!)
+- Update `Enable-PSRemoting` so configuration name for Preview releases (#7202)
+- Improve performance on JSON to PSObject conversion (#7482) (Thanks @powercode!)
+- Fix error message for `Add-Type` when `-AssemblyName` with wildcard is not found (#7444)
+- Make Native globbing on Unix return an absolute path when it is given an absolute path (#7106)
+- Improve the performance of `Group-Object`  (#7410) (Thanks @powercode!)
+- Remove one unneeded verbose output from `ConvertTo-Json` (#7487) (Thanks @devblackops!)
+- Enable `Get-ChildItem` to produce `Mode` property even if cannot determine if hardlink (#7355)
+
+### Code Cleanup
+
+- Remove empty XML comment lines (#7401) (Thanks @iSazonov!)
+- Cleanup Docker files (#7328)
+- Correct the comment for `WSManReceiveDataResult.Unmarshal` (#7364)
+- Format Utility `csproj` with updated `codeformatter` (#7263) (Thanks @iSazonov!)
+- Bulk update format for files in Management folder with `codeformatter` (#7346) (Thanks @iSazonov!)
+- Cleanup: replace `Utils.FileExists()/DirectoryExists()/ItemExists()` with DotNet methods (#7129) (Thanks @iSazonov!)
+- Update `Utils.IsComObject` to use `Marshal.IsComObject` since CAS is no longer supported in DotNet Core (#7344)
+- Fix some style issues in engine code (#7246) (Thanks @iSazonov!)
+
+### Test
+
+- Use `-BeExactly` and `-HaveCount` instead of `-Be` in `BugFix.Tests.ps1` (#7386) (Thanks @sethvs!)
+- Use `-BeExactly` and `-HaveCount` instead of `-Be` in `TabCompletion.Tests.ps1` (#7380) (Thanks @sethvs!)
+- Update CI scripts to support running tests for experimental features (#7419)
+- Use `-HaveCount` instead of `-Be` in `Where-Object.Tests.ps1` (#7379) (Thanks @sethvs!)
+- Fix ThreadJob tests so that they will run more reliably (#7360)
+- Make logging tests for macOS pending (#7433)
+
+### Build and Packaging Improvements
+
+- Update Build script owners (#7321)
+- Make `MUSL` NuGet package optional (#7316)
+- Enable `pwsh-preview` to work on Windows (#7345)
+- Fix SDK dependencies
+- Add back the `powershell-core` NuGet source for hosting tests
+- Fix typo in environment checker (#7547 & #7549)
+- Only remove the revision if it is `0` from module version when restoring modules (#7538)
+- Update `WCF` and `NJsonSchema` NuGet packages to latest released patch version (#7411) (Thanks @bergmeister!)
+- Update PowerShell to reference `psrp.1.4.5-0` on Unix build (#7350)
```

Comment:
This changed to used `Microsoft.PowerShell.Native` package. Should this be in the CHANGELOG?